src/vision/oak/camera_oak.py

```diff
@@ -0,0 +1,81 @@
+import depthai as dai
+import open3d as o3d
+import numpy as np
+from vision.oak.imu_tracker import ImuTracker
+from vision.oak.camera_hendler import CameraHendler
+
+
+class CameraOAK:
+    """class for init camera and get data from it."""
+
+    def __init__(self, config: dict, visualize: bool = False):
+        """ Initialize the CameraOAK class.
+        Args:
+            config (dict): The configuration dictionary.
+        """
+        self.handler = CameraHendler(config)
+        self.device = dai.Device(self.handler.pipeline)
+        self.base_time = None
+        self.imu_tracker = ImuTracker()
+        self.visualize = visualize
+        if visualize:
```

Comment:
nie można przenieść wsm tego do osobnej metody i wywoływać to tylko w __init__?
```python
def __init__(self, config: dict, visualize: bool = False):
        self.handler = CameraHendler(config)
        self.device = dai.Device(self.handler.pipeline)
        self.base_time = None
        self.imu_tracker = ImuTracker()
        self.visualize = visualize
        self.init_visualizer()

    def init_visualizer(self) -> None:
        if self.visualize:
            self.vis = o3d.visualization.Visualizer()
            self.vis.create_window()
            self.line_points = []
            self.line_set = o3d.geometry.LineSet()
            self.vis.add_geometry(self.line_set)
```

Reply:
done